src/components/input/input.module.css

```diff
@@ -0,0 +1,42 @@
+.label {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  gap: 4px;
+
+  font-size: 16px;
+  color: var(--color-font-secondary);
```

Comment:
bad contrast ration

<img width="733" alt="image" src="https://user-images.githubusercontent.com/23063254/212546392-153b9f61-e5ee-45d4-be6d-635769f99576.png">

Reply:
contrast ratio > 5 is good?